sunbeam-python/sunbeam/jobs/steps.py

```diff
@@ -322,3 +330,98 @@ def run(self, status: Optional[Status] = None) -> Result:
             return Result(ResultType.FAILED, str(e))
 
         return Result(ResultType.COMPLETED)
+
+
+def proxy_questions():
+    return {
+        "proxy_required": PromptQuestion(
```

Comment:
You can use a `ConfirmQuestion` not to handle the y/n specifically here

Reply:
Done